test/node/reducers/vis-state-test.js

```diff
@@ -4924,8 +4924,11 @@ test('#visStateReducer -> LOAD_FILES', async t => {
   // test LOAD_FILE_TASK success
   const [task2, ...more2] = drainTasksForTesting();
   t.equal(more2.length, 0, 'should ceate 1 task');
-  t.equal(task2.type, 'UNWRAP', 'should return an UNWRAP task');
-  t.ok(task2.payload instanceof Promise, 'task 2 payload should be a Promise');
+  t.equal(task2.type, 'Chain(DELAY)', 'should return an Chain(DELAY) task');
+  t.equal(task2.payload, 10, 'should delay 10 millisecond of the next task');
+  // TODO: FIXME task2 is a chain task with a 10 millisecond delay and a UNWRAP_TASK task
+  // Here the payload of UNWRAP_TASK is a Promise. However, it seems we cannot test it here:
+  // t.ok(task2.payload instanceof Promise, 'task 2 payload should be a Promise');
```

Comment:
@heshan0131 Hi Shan, do you know how to test the chained task? Thank you!

Reply:
It may be easier to just wrap the batch iterator returned from loaders.gl in a new iterator that sleeps say 50ms between each batch. In fact I think there already is a `batchDebounceMs` option in loaders.gl that inserts such waits.

Reply:
Thanks, @ibgreen! That sounds reasonable. I just tested: adding a delay between each batch works the same. See this draft PR here: https://github.com/visgl/loaders.gl/pull/2836 But I am not sure how to pass the `batchDebounceMs` option to the `parseArrowInBatches()` function. Can you help to take a look? Thanks!

Reply:
Looking around but getting lost in the code. Is loading going through the ArrowLoader in `file-handler.ts`? The general pattern would be to just add it to the loader options root:

```ts
load(url, [loaders...], {
   batchDebounceMs: 100,
   csv: {...}
});
```

Reply:
Thanks, Ib! I can add the option in kepler's file-handler.ts, like this:
<img width="377" alt="Screenshot 2023-12-11 at 4 13 14 PM" src="https://github.com/keplergl/kepler.gl/assets/2423887/8a5b0c2a-aa62-4f42-9e02-b4e442c690f0">

Then, how can I pass it down to the `parseArrowInBatches()` function in loaders.gl? Should I add option parameter for `parseInBatches` here  Line 60?: 
<img width="623" alt="Screenshot 2023-12-11 at 4 16 01 PM" src="https://github.com/keplergl/kepler.gl/assets/2423887/2069dc16-8587-4916-a8ba-005b1fed3ea5">

Reply:
I think I got it workign in https://github.com/visgl/loaders.gl/pull/2836. I will update the loader.gl version for this PR once it's been merged and tagged. Thanks for your help, @ibgreen !

Reply:
TODO: need a new release of loader.gl to complete this PR.